test/tokens/ERC1155.t.sol

```diff
@@ -163,9 +163,15 @@ contract ERC1155Test is Test, ERC1155Recipient, FuzzingUtils{
     /// @dev Setup the testing environment.
     function setUp() public {
         string memory wrappers = vm.readFile("test/tokens/mocks/ERC1155Wrappers.huff");
+        HuffDeployer.config().with_args(bytes.concat(abi.encode("Token"), abi.encode("TKN")));
         token = ERC1155(HuffDeployer.deploy_with_code("tokens/ERC1155", wrappers));
```

Comment:
Doing it in one line with `.config().with_args` or `.deploy_with_code_args` creates some really weird bugs (probably in from HuffConfig). I will investigate later.